dds/idl/be_global.cpp

```diff
@@ -854,11 +859,18 @@ ExtensibilityKind BE_GlobalData::extensibility(AST_Decl* node) const
       builtin_annotations_["::@extensibility"]);
   ExtensibilityKind value;
   if (!extensibility_annotation->node_value_exists(node, value)) {
-    value = default_extensibility_;
+    value = default_extensibility;
+    has_annotation = false;
   }
   return value;
 }
 
+ExtensibilityKind BE_GlobalData::extensibility(AST_Decl* node) const
+{
+  bool has_annotation = false;
+  return extensibility(node, default_extensibility_, has_annotation);
```

Comment:
Since `has_annotation` isn't read after it's written, should there be another overload like `extensibility(node, default_extensibility_)` that ignores has_annotation?